src/main/kotlin/uk/gov/dwp/dataworks/controllers/ConnectionController.kt

```diff
@@ -63,8 +63,8 @@ class ConnectionController {
     ])
 
     @PostMapping("/deployusercontainers")
-    fun launchTask(@RequestHeader("Authorisation") userName: String, @RequestBody requestBody: DeployRequest): String {
-        return handleRequest(userName, requestBody)
+    fun launchTask(@RequestHeader("Authorisation") userName: String, @RequestHeader("cognito:groups") cognitoGroups: List<String>, @RequestBody requestBody: DeployRequest): String {
```

Comment:
Why add this as a header instead of to the Request body?

Reply:
It is added in the header to be as close to the actual working as possible - the JWT will be sent through as part of the header.